controllers/session/session_controller.go

```diff
@@ -270,6 +264,11 @@ func (r *ReconcileSession) Reconcile(c context.Context, request reconcile.Reques
 						session.Status.Hosts = splitAndUnique(session.Status.Hosts, modified.Prop["hosts"])
 					}
 				}
+				if modified.Success {
+					session.Status.Readiness.Components.SetReady(modified.Kind + "/" + modified.Name)
+				} else {
+					session.Status.Readiness.Components.SetUnReady(modified.Kind + "/" + modified.Name)
```

Comment:
Shouldn't it be `SetUnready`?

Reply:
it could be...

Reply:
I mean `unready` is a word :)

Reply:
fixed https://github.com/maistra/istio-workspace/pull/875/commits/09e0326e8adbf938d417565697d7161f702ee273